frontend/apps/filemanager/filemanager.lua

```diff
@@ -248,6 +261,28 @@ function FileManager:init()
     self:handleEvent(Event:new("SetDimensions", self.dimen))
 end
 
+function FileManager:fileInformation(file)
+    --local book_stats = {}
+    local file_mode = lfs.attributes(file, "mode")
+    if file_mode == "file" then
+        local book_stats = DocSettings:open(file):readSetting('stats')
+        if book_stats ~= nil then
+            return FileManagerHistory:printFileInformation(book_stats)
+        else
+            local document = DocumentRegistry:openDocument(file)
+            ReaderUI:new{
+                dimen = Screen:getSize(),
+                document = document,
+            }
+            book_stats = document:getProps()
+            book_stats.pages = document:getPageCount()
+            return FileManagerHistory:printFileInformation(book_stats)
```

Comment:
This method name is a little bit misleading because it's not actually printing anything. Perhaps rename it to `buildBookInformationTable` or `formatBookInformation`.